src/controller/tests/logout.controller.test.ts

```diff
@@ -0,0 +1,28 @@
+import Logout from '../logout.controller'
+import { refreshToken } from '../../utils/cookie.utils'
```

Comment:
```suggestion

import { refreshToken } from '../../utils/cookie.utils'
import Testing from '../../utils/testing.utils'
```

This is some of the most nitpicky shit ever I'm so sorry 🤣